src/main/java/de/hpi/bpt/argos/eventProcessing/mapping/EventEntityMapperImpl.java

```diff
@@ -0,0 +1,122 @@
+package de.hpi.bpt.argos.eventProcessing.mapping;
+
+
+import de.hpi.bpt.argos.common.Observable;
+import de.hpi.bpt.argos.common.ObservableImpl;
+import de.hpi.bpt.argos.eventProcessing.EventCreationObserver;
+import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
+import de.hpi.bpt.argos.storage.dataModel.attribute.Attribute;
+import de.hpi.bpt.argos.storage.dataModel.attribute.type.TypeAttribute;
+import de.hpi.bpt.argos.storage.dataModel.entity.Entity;
+import de.hpi.bpt.argos.storage.dataModel.event.Event;
+import de.hpi.bpt.argos.storage.dataModel.event.type.EventType;
+import de.hpi.bpt.argos.storage.dataModel.mapping.EventEntityMapping;
+import de.hpi.bpt.argos.storage.dataModel.mapping.MappingCondition;
+import de.hpi.bpt.argos.util.LoggerUtilImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+/**
+ * {@inheritDoc}
+ * This is the implementation.
+ */
+public class EventEntityMapperImpl extends ObservableImpl<EventMappingObserver> implements EventEntityMapper {
+	private static final Logger logger = LoggerFactory.getLogger(EventEntityMapperImpl.class);
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void setup(Observable<EventCreationObserver> eventReceiver) {
+		eventReceiver.subscribe(this);
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void onEventCreated(EventEntityMappingStatus mappingStatus,
+			EventType eventType, List<TypeAttribute> eventTypeAttributes, Event event, List<Attribute> eventAttributes) {
+
+		if (mappingStatus.isMapped()) {
```

Comment:
Since this is implemented as an observer-pattern, there might be other classes which are interested in the creation-process. Thus, there might be a custom mapping implementation. In such cases this if-statement becomes relevant.